python-sdk/src/astro/dataframes/load_options.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+from attr import asdict, define
+from pandas._typing import DtypeArg
+
+from astro.options import LoadOptions
+
+
+@define
+class PandasLoadOptions(LoadOptions):
+    @property
+    def to_dict(self):
+        return dict(filter(lambda x: x[1], asdict(self).items()))
```

Comment:
@pankajastro Can we use asdict(self) instead of this?

Reply:
fixed it. remove to_dict and using asdict